Update to adjust expando column width to fit text on one line.

andrethomas6 opened the pull request:
Update to adjust expando column width to fit text on one line.

sanjuroj reviewed (approved):
Not that you're changing anything related to accessibility, but since you're here, did you check to see if the expando is keyboard accessible?  Do we need to handle Enter commands as well?

andrethomas6 commented:
Hey @sanjuroj I added aria labels, tab and escape key functionality for the IPTT accordion. I also added the ability to escape out of three modals with the escape key.

sanjuroj commented on scss/_iptt_table.scss:
```diff
@@ -126,10 +126,13 @@ td.iptt-level-row, td.ip-level-row {
     color: $blue;
     cursor: pointer;
     border-right: none;
+    div { // Will adjust the width be long enouugh to fit text on one line.
```
typo on "enouugh"

sanjuroj reviewed (approved):
Looks good, thanks for making the updates!
